src/modules/flash/producer/flash_producer.cpp

```diff
@@ -153,6 +157,22 @@ std::mutex& get_global_init_destruct_mutex()
     return m;
 }
 
+std::wstring url_from_path(std::wstring in)
+{
+    DWORD        out_length = INTERNET_MAX_URL_LENGTH * 2 + 4;
```

Comment:
Why +4?

Reply:
This is to allow for null termination of wstring.